packages/legend-query/src/stores/QueryBuilderLambdaProcessor.ts

```diff
@@ -385,11 +385,20 @@ export class QueryBuilderLambdaProcessor
   ): void {
     const functionName = valueSpecification.functionName;
     if (matchFunctionName(functionName, SUPPORTED_FUNCTIONS.GET_ALL)) {
+      const _class = valueSpecification.genericType?.value.rawType;
+      let acceptedNoOfParameters = 1;
+      if (_class instanceof Class && _class.stereotypes.length !== 0) {
+        acceptedNoOfParameters = valueSpecification.parametersValues.length;
```

Comment:
we can just set this to 2 here then do the check of the `valueSpecification.parametersValues.length === acceptedNoOfParameters` below